### PR TITLE
Pf/style remediation 1

### PR DESCRIPTION
**Pull request by firefox0102**
First style remediation updates

### CODE REVIEW

**Comment by laurenlangley** [comment]
@firefox0102 I ran the style remediation build but my credentials were not working so I can only comment on Sign In page:
- [x] footer height is better but footer should still be fixed to bottom
- [x] social icons on footer are not visible

File: src/App.vue [hunk 1]
```diff
@@ -76,9 +74,9 @@ body {
   overflow-y: auto;
 
   &__body {
-    min-height: calc(100vh - 64px - 140px);
+    min-height: calc(100vh - 64px - 106px);
```
Comment (firefox0102) [inline src/App.vue:77]: I need to make this a variable

File: src/components/admin/AddCollectionSite.vue [hunk 2]
```diff
@@ -104,3 +102,15 @@ export default {
   }
 }
 </script>
+
+<style lang="scss">
+@import "../../scss/colors";
+
+.add-header-toolbar {
```
Comment (firefox0102) [inline src/components/admin/AddCollectionSite.vue:144]: This can be removed

File: src/components/admin/AddCollectionSite.vue [hunk 3]
```diff
@@ -82,7 +79,8 @@ export default {
         logbookAbbv: '',
         longitude: '',
         stationName: '',
-        storetName: ''
+        storetName: '',
+        collectionSiteId: null // TODO set the ID when you save it
```
Comment (firefox0102) [inline src/components/admin/AddCollectionSite.vue]: This TODO can go

File: src/components/admin/CollectionSites.vue [hunk 4]
```diff
@@ -179,8 +179,13 @@ export default {
   components: {
     'add-collection-site': AddCollectionSite
   },
-  firebase: {
-    collectionSites: collectionSitesRef
+  firebase () {
+    return {
+      collectionSites: collectionSitesRef,
+      activeSites: {
```
Comment (firefox0102) [inline src/components/admin/CollectionSites.vue:185]: This is crappy, we should have a data structure for saving this high level info maybe?

File: src/components/admin/LogData.vue [hunk 5]
```diff
@@ -351,17 +394,17 @@ export default {
         this.newLogData.collectionSite = null
 
         // Set logbook number
-        this.newLogData.logbookNumber = this.getLastLogbookNumber()
+        // this.newLogData.logbookNumber = this.getLastLogbookNumber()
```
Comment (firefox0102) [inline src/components/admin/LogData.vue]: Remove this commented code

**Pull request merged**